api/handlers/groups.go

```diff
@@ -18,8 +18,15 @@ type groupsHandler struct {
 }
 
 func (gH groupsHandler) list(w http.ResponseWriter, r *http.Request) {
-	log.Trace("hosts path called")
 	vars := mux.Vars(r)
+	var parents bool = false
```

Comment:
```suggestion
	var parents bool
```
by default boolean values are `false`